app/controllers/issues/evidence_controller.rb

```diff
@@ -0,0 +1,56 @@
+class Issues::EvidenceController < AuthenticatedController
+  include ContentFromTemplate
+  include DynamicFieldNamesCacher
+  include MultipleDestroy
+  include ProjectScoped
+
+  before_action :set_issues, only: [:index, :new]
+  before_action :set_affected_nodes, only: :index
+  before_action :set_columns, only: :index
+
+  def index
+    render layout: false
+  end
+
+  def new
+    @nodes_for_add_evidence = current_project.nodes.user_nodes.order(:label)
+
+    @template_content = template_content if params[:template]
+  end
+
+  private
+
+  def set_columns
+    default_field_names = ['Affected', 'Title', 'Created', 'Updated'].freeze
```

Comment:
We can move 'Created' and 'Updated' of the default-show list (consistent with Issues) and use the already available Evidence#local_field 'Label' instead of 'Affected'

Reply:
Done